src/Hl7.Fhir.Specification.Tests/Hl7.Fhir.Specification.Tests.csproj

```diff
@@ -52,7 +52,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
     <Content Include="TestData\ResourcesInSubfolder.zip">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Are you sure? This can delay the compilation.

Reply:
Oh, that shouldn't have been changed. I'll set it back to PreserveNewest. I had some trouble with tests relying on that file failing so I changed the setting, but it should never have been committed